### PR TITLE
Functional tests - Refacto 'Orders > Delivery slips' tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Refacto delivery slips tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | no need
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25104)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/02_orders/04_deliverySlips/02_deliverySlipOptions/03_enableDisableProductImage.js [hunk 1]
```diff
@@ -61,89 +69,130 @@ describe('Test enable/disable product image in delivery slips', async () => {
...
+        it('should go to login page', async function () {
+          await testContext.addContextItem(this, 'testIdentifier', `goToLoginPageFO${index}`, baseContext);
+
+          await homePage.goToLoginPage(page);
+          const pageTitle = await foLoginPage.getPageTitle(page);
+          await expect(pageTitle, 'Fail to open FO login page').to.contains(foLoginPage.pageTitle);
+        });
+
+        it('should sign in with default customer', async function () {
+          await testContext.addContextItem(this, 'testIdentifier', `sighInFO${index}`, baseContext);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/04_deliverySlips/02_deliverySlipOptions/03_enableDisableProductImage.js]:
```suggestion
          await testContext.addContextItem(this, 'testIdentifier', `signInFO${index}`, baseContext);
```

File: tests/UI/campaigns/functional/BO/02_orders/04_deliverySlips/02_deliverySlipOptions/03_enableDisableProductImage.js [hunk 2]
```diff
@@ -61,89 +69,130 @@ describe('Test enable/disable product image in delivery slips', async () => {
...
           // Payment step - Choose payment step
           await checkoutPage.choosePaymentAndOrder(page, PaymentMethods.wirePayment.moduleName);
 
           // Check the confirmation message
           const cardTitle = await orderConfirmationPage.getOrderConfirmationCardTitle(page);
           await expect(cardTitle).to.contains(orderConfirmationPage.orderConfirmationCardTitle);
+        });
+
+        it('should sign out from FO', async function () {
+          await testContext.addContextItem(this, 'testIdentifier', `sighOutFO${index}`, baseContext);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/02_orders/04_deliverySlips/02_deliverySlipOptions/03_enableDisableProductImage.js]:
```suggestion
          await testContext.addContextItem(this, 'testIdentifier', `signOutFO${index}`, baseContext);
```

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

File: tests/UI/campaigns/functional/BO/02_orders/04_deliverySlips/02_deliverySlipOptions/03_enableDisableProductImage.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/02_orders/04_deliverySlips/02_deliverySlipOptions/03_enableDisableProductImage.js]: Done thx :+1:

File: tests/UI/campaigns/functional/BO/02_orders/04_deliverySlips/02_deliverySlipOptions/03_enableDisableProductImage.js (hunk 2 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/02_orders/04_deliverySlips/02_deliverySlipOptions/03_enableDisableProductImage.js]: Done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — CHANGES_REQUESTED** [review]
## WAIT FOR 1780 RELEASE

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh & @boubkerbribri

**Pull request merged**